org.gradle.toolingapi/.project

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
```

Comment:
❌ I'm a bit confused by the whole setup. Why is the TAPI jar checked in? Shouldn't that be the job of the build to set up the project?

Reply:
The jar is only needed to make the Eclipse workspace compile successfully. Without it, we'd need to execute Gradle tasks when setting up the dev environment.